notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -1,1140 +1,1273 @@
 {
```

Comment:
<p>Why is there a link here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>